crates/holochain_zome_types/src/zome_io.rs

```diff
@@ -231,6 +231,13 @@ pub enum ZomeCallResponse {
     /// A countersigning session has failed to start.
     CountersigningSession(String),
 }
+
+impl std::fmt::Display for ZomeCallResponse {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_fmt(format_args!("{:?}", self))
```

Comment:
```suggestion
        write!(f, "{:?}", self)
```

little more succinct